### PR TITLE
feat(ori-1936): allow environment variables to be set

### PR DESCRIPTION
**Pull request by don1989**
## Why
- Allow api key, secret, base url, env to be set via process.env. vars

### How
- Add environment variables support
- Update readme

### How Has This Been Tested?
- Locally

### Checklist
Closes ORI-1933

### CODE REVIEW

File: test/e2e/.env.test [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+TEST_APP_USER_EMAIL=user_email_1@getoriginal.com
+TEST_APP_USER_USER_EXTERNAL_ID=user_external_id_1
+TEST_COLLECTION_UID=760475580039
+TEST_REWARD_UID=129215903628
+TEST_ASSET_UID=738364220773
+TEST_TRANSFER_TO_WALLET_ADDRESS=0x87e1b883d5bde55c9e82d3cecab67315af80828b
+TEST_TRANSFER_TO_USER_UID=982915059544
+TEST_ALLOCATION_UID=330661354307
+TEST_CLAIM_UID=204158886400
+TEST_CLAIM_TO_ADDRESS=0xd8da6bf26964af9d7eed9e03e53415d37aa96045
```
Comment (Alekso91) [inline test/e2e/.env.test:17]: are we sure we want those to be public?

Comment (don1989) [reply test/e2e/.env.test:17]: Probably for the best indeed. We can create a file with the values stored in our notion page for local testing.

Comment (don1989) [reply test/e2e/.env.test]: Deleted the file. We can just manually add the [file](https://www.notion.so/getoriginal/SDK-Test-Env-Variables-1354ea3c3b1c80e185efda712d7828d1) locally.

**Review by Alekso91 — APPROVED** [review]
lgtm

**Pull request merged**